eth/protocols/eth/peer.go

```diff
@@ -331,63 +350,110 @@ func (p *Peer) SendBlockHeaders(headers []*types.Header) error {
 	return p2p.Send(p.rw, BlockHeadersMsg, BlockHeadersPacket(headers))
 }
 
-// SendBlockBodies sends a batch of block contents to the remote peer.
-func (p *Peer) SendBlockBodies(bodies []*BlockBody) error {
-	return p2p.Send(p.rw, BlockBodiesMsg, BlockBodiesPacket(bodies))
+func (p *Peer) ReplyBlockHeaders(id uint64, headers BlockHeadersPacket) error {
```

Comment:
Please add method docs for all `Reply*` functions

Reply:
I'm a bit reluctant about the argument type `headers BlockHeadersPacket`. Imho we should keep the `[]*types.Header` type as with the Send* method. The point of SendXXX and ReplyXXX is to package up the data into the network packet. We shouldn't package it in advance (I know it's a type alias, still, that's a detail)